src/RestControllers/FHIR/FhirDocumentRestController.php

```diff
@@ -74,7 +73,7 @@ public function downloadDocument($documentId, $userId): ResponseInterface
                 // we just continue as we still wanto to reject the response
                 $this->logger->error(
                     "FhirDocumentRestController->downloadDocument() Failed to delete document with id",
-                    ['document' => $documentId, 'user' => $userId, 'exception' => $exception->getMessage()]
+                    ['document' => $documentId, 'username' => $_SESSION['authUser'], 'exception' => $exception->getMessage()]
```

Comment:
@adunsulag ,
just making sure ok with this PR. I am not sure how the $userId was working here in the call by _rest_routes.inc.php since that function should be using username and not the user id or user uuid. These changes will imply that the user set in the session will be used. Would it make sense to completely remove these username logs since no longer bringing this in as a parameter?

Reply:
good research and question